art/attacks/evasion/carlini.py

```diff
@@ -515,10 +516,10 @@ class CarliniLInfMethod(EvasionAttack):
         "targeted",
         "learning_rate",
         "max_iter",
-        "max_halving",
-        "max_doubling",
-        "eps",
-        "batch_size",
+        "decrease_factor",
```

Comment:
If you change parameter names here, maybe also consider changing them for the L0 and L2 counterparts.

Reply:
Good point. I have considered the L2 method but it is based on a different algorithm/optimisation and requires different inputs.